### ISSUE TITLE
using getDate() in date parser can result in day being off by one

### ISSUE THREAD

**Issue by dylang** [excitement-engineer/graphql-iso-date#26]
```js
> new Date('10/30/1975').getDate()
30
> new Date('1975-10-30').getDate()
29
```
I'm on EDT (US eastern daylight time). My server is not, so it gets `30` for both.

**Comment by dylang** [comment]
The problem might be using `.getDate()` which seems to be relative to where the code is running.

**Comment by dylang** [comment]
After further investigation, it seems that using `.getUTCDate()` instead of `getDate()` will fix this issue. 

See [formatter.js#L81-L88](https://github.com/excitement-engineer/graphql-iso-date/blob/master/src/utils/formatter.js#L81-L88).

**Comment by excitement-engineer** [comment]
Thanks for the bug report! In addition to the day of the month, all of the other different date parts have to be set to UTC as well, so the code will have to look something like this.

```js
export const serializeDate = (date: Date): string => {
  let month = String(date.getUTCMonth() + 1)
  let day = String(date.getUTCDate())
  const year = String(date.getUTCFullYear())

  if (month.length < 2) month = '0' + month
  if (day.length < 2) day = '0' + day
  return [year, month, day].join('-')
}
```

I will update the library shortly with this bug fix:)

**Comment by excitement-engineer** [comment]
This has been fixed in version 3.1.1. Thanks for pointing this out!
